lib/player.js

```diff
@@ -3761,6 +3798,51 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return expected;
   }
 
+  /**
+   * Return a list of chapters tracks.
+   *
+   * @return {!Array.<shaka.extern.Track>}
+   * @export
+   */
+  getChaptersTrackList() {
```

Comment:
For consistency with getVariantTracks, getTextTracks, please remove "List" from this method name.

Reply:
Done!